src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/HearingDocumentController.java

```diff
@@ -61,6 +68,10 @@
         Map<String, Object> caseData = caseDetails.getData();
         caseData.putAll(service.generateHearingDocuments(authorisationToken, caseDetails));
 
+        if (hasSolicitorAgreedToReceiveEmails(caseData)) {
+            notificationService.sendPrepareForHearingEmail(callback);
+        }
+
```

Comment:
I would imagine this requires a unit test in the HearingDocumentControllerTest.java file, however the scenario is also covered in the Notification Controller. Is this needless repetition?

Reply:
Yep should still have unit tests to cover this - use something like
verifyNoInteractions(notificationService)....

Reply:
added